netutils/mac.py

```diff
@@ -136,3 +136,29 @@ def mac_normalize(mac: str) -> str:
         if char in mac:
             mac = mac.replace(char, "")
     return mac
+
+
+@_valid_mac
+def get_oui(mac: str) -> t.Optional[str]:
+    """Returns the company name for a given mac as defined by the IEEE.
+
+    Args:
+        mac (str): A MAC address in string format that matches one of the defined regex patterns.
```

Comment:
type hints are discovered now and we now remove them from the docstrings.

Reply:
Fixed. Thanks Jeff.